.github/workflows/docker.yml

```diff
@@ -0,0 +1,44 @@
+name: Build Docker
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - '*'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to Docker Hub
```

Comment:
I think ghcr.io and docker hub aren't the same?

Reply:
Fixed, it was a copy paste problem :wink: